src/components/Pagination.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { MouseEvent } from 'react';
+import * as styles from './Pagination.css';
+
+type Props = {
+  articlesCount: number;
+  handlePagination: (e: MouseEvent<HTMLButtonElement>) => any;
```

Comment:
(마이너) 컴포넌트에서 props로 이벤트 핸들러 함수를 받을때는 on- prefix를 일반적으로 사용하는 것 같은데요! 괜찮다고 생각하시면 반영해주시고 아니면 넘어가셔도 괜찮을 것 같아요~

Reply:
추가로 반환값이 없는 함수의 경우에는 리턴 타입을 any 대신에 void로 잡아주시면 되어요!

```suggestion
  handlePagination: (e: MouseEvent<HTMLButtonElement>) => void;
```